apps/src/sites/studio/pages/code-studio.js

```diff
@@ -44,6 +44,7 @@ window.dashboard.videos = require('@cdo/apps/code-studio/videos');
 window.dashboard.assets = require('@cdo/apps/code-studio/assets');
 window.dashboard.pairing = require('@cdo/apps/code-studio/pairing');
 window.dashboard.teacher = require('@cdo/apps/code-studio/teacher');
+window.dashboard.project = require('@cdo/apps/code-studio/initApp/project');
```

Comment:
I wonder how noisy this would be, as I believe there are a bunch of places where we know we use `window.dashboard.project` directly (i.e. in haml files).